integration-tests/emulated/tests/bridges/bridge-hub-polkadot/src/tests/snowbridge.rs

```diff
@@ -459,6 +461,184 @@ fn send_token_from_ethereum_to_asset_hub() {
 	});
 }
 
+/// Tests sending ether from Ethereum to Asset Hub and back to Ethereum
+#[test]
+fn send_eth_asset_from_asset_hub_to_ethereum() {
```

Comment:
I can imagine to create some generic helper function:

```
fn send_token_from_ethereum_to_asset_hub_and_back_works(token: Location) {
   // copy the code below
}
```

and then reuse for test-cases, something like:
```
#[test]
fn send_weth_from_ethereum_to_asset_hub() {
    send_token_from_ethereum_to_asset_hub_and_back_works(WETH)
}

#[test]
fn send_eth_from_ethereum_to_asset_hub() {
    let ether_location: Location = (Parent, Parent, EthereumNetwork::get()).into();
    send_token_from_ethereum_to_asset_hub_and_back_works(ether_location)
}
```

Reply:
Addressed in https://github.com/polkadot-fellows/runtimes/pull/548/commits/798229c61137f4f063c9a059e03b9259a8ac24b6